src/shared/components/svg-icon.js

```diff
@@ -34,10 +51,12 @@ export default function SvgIcon({
   }
   const markup = { __html: iconRegistry[name] };
 
-  const element = useRef();
+  const element = /** @type {Ref<HTMLElement>} */ (useRef());
```

Comment:
Were TS errors clear enough to get you here quickly? I'm trying to envision my workflow if I were trying to add typechecking to this module and encountered a `useRef()`...

Reply:
Yes, although I will admit that I have quite a bit of experience working with TS and similar tools outside of Hypothesis.